notifier/processor_create_test.go

```diff
@@ -216,13 +217,13 @@ func testProcessorCreate(t *testing.T) {
 			}, nil
 		},
 	}
-	count := 0
+	count := uint64(0)
```

Comment:
This looks good, p.create was updated recently to do work inside go routines, and this must have been missed.